sdk/python/kfmd/metadata.py

```diff
@@ -41,8 +45,10 @@ def __init__(self,
     self.name = name
     self.description = description
     self.labels = labels
-    self._user = "" #TODO: get user from env
-    self._id = ""
+
+    config = Configuration()
+    config.host = backend_url_prefix
+    self._client = MetadataServiceApi(ApiClient(config))
 
 class Run(object):
```

Comment:
Should we add the API to create an execution for the run?

Reply:
If this is one of the intermediate PRs towards the complete API SDK support, then LGTM

Reply:
This PR won't complete the SDK. We can expose more method if needed.